Create trabalho-final-AugustoCravo

AugustofCravo opened the pull request:
Professor, adicionei um pull  request que fiz para o The Algorithm. Mas ainda tenho dúvidas se isso conta como colaboração. Quando possível, gostaria q o senhor pudesse confirmar. Obrigado.

AugustofCravo committed:
Create trabalho-final-AugustoCravo

Professor, adicionei um pull  request que fiz para o The Algorithm. Mas ainda tenho dúvidas se isso conta como colaboração. Quando possível, gostaria q o senhor pudesse confirmar. Obrigado.

gustavopinto commented:
Oi @AugustofCravo, 

faça a modificação que foi sugerida na revisão do seu PR (https://github.com/TheAlgorithms/Python/pull/936) e então isso contará como atividade pequena.

AugustofCravo committed:
Update trabalho-final-AugustoCravo

Atualizada a primeira tarefa e adicionado mais duas outras de pequena contribuição.

AugustofCravo commented:
Professor, 

Fiz a modificação que o senhor pediu e adicionai mais outras duas tarefas.

gustavopinto commented:
Coloca uma tabela com o link para as contribuições e os pontos pretendidos?

AugustofCravo committed:
Update trabalho-final-AugustoCravo

Professor, ao final do documento adicionei link para tabela com as informações que o senhor pediu.

AugustofCravo commented:
Professor, ao final do documento, adicionei link para tabela que o senhor pediu.
